providers/site.rb

```diff
@@ -98,6 +101,30 @@
       @new_resource.updated_by_last_action(true)
     end
 
+    if @new_resource.log_directory && is_new_log_directory
+      cmd = "#{appcmd(node)} set site \"#{@new_resource.site_name}\""
+      cmd << " /logFile.directory:#{@new_resource.log_directory}"
```

Comment:
I presume this is a windows path.  If so, could you use windows_cleanpath() like the /physicalPath code above?